src/test/resources/findex/expected_db_uids.json

```diff
@@ -1,32 +1,32 @@
 [
-  "3e3e3e3e3e3e3e3e3e3e3e3e3e3e3e3e",
-  "53535353535353535353535353535353",
-  "42424242424242424242424242424242",
-  "06060606060606060606060606060606",
-  "45454545454545454545454545454545",
-  "51515151515151515151515151515151",
-  "48484848484848484848484848484848",
-  "61616161616161616161616161616161",
-  "23232323232323232323232323232323",
-  "57575757575757575757575757575757",
-  "40404040404040404040404040404040",
-  "18181818181818181818181818181818",
-  "2c2c2c2c2c2c2c2c2c2c2c2c2c2c2c2c",
-  "08080808080808080808080808080808",
-  "31313131313131313131313131313131",
-  "05050505050505050505050505050505",
-  "4e4e4e4e4e4e4e4e4e4e4e4e4e4e4e4e",
-  "15151515151515151515151515151515",
-  "12121212121212121212121212121212",
-  "3d3d3d3d3d3d3d3d3d3d3d3d3d3d3d3d",
-  "38383838383838383838383838383838",
-  "26262626262626262626262626262626",
-  "09090909090909090909090909090909",
-  "0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f",
-  "39393939393939393939393939393939",
-  "14141414141414141414141414141414",
-  "47474747474747474747474747474747",
-  "2f2f2f2f2f2f2f2f2f2f2f2f2f2f2f2f",
-  "54545454545454545454545454545454",
-  "56565656565656565656565656565656"
-]
+  4,
```

Comment:
well I regret the good old `expected_db_uids.json` which was the same for the rust lib and python lib and then convenient for regression tests

Reply:
Didn't see them in the Rust lib but I will change them in Python too because for the compact we need IDs with a corresponding value in database (to be able to remove some users in database, compact, and check the following search). These IDs were hardcoded for the order of the JSON input and not saved inside the database. The creation of these IDs were quite a lot of code too in Java